.env.example

```diff
@@ -1,27 +0,0 @@
-APP_WWW=http://localhost
```

Comment:
We should reinstate this file. It is used to tell people the structure of the expected .env file. Thanks